lib/templates/bootstrap/select.ios.js

```diff
@@ -43,8 +46,8 @@ class CollapsiblePickerIOS extends React.Component {
 
     const height = (this.state.isCollapsed) ? 0 : UIPICKER_HEIGHT;
     return (
-      <View>
-        <TouchableOpacity style={touchableStyle}
+      <View style={[pickerContainer, (!this.state.isCollapsed) ? pickerContainerOpen : {}]}>
```

Comment:
Utilising both removes the need for duplicate code and allows the styling to work in a manner similar to CSS.

For instance, if all you wanted to do was change the `borderColor` for the for open state, if you were to use one or the other, the style declaration would like this:

```javascript
pickerContainer: {
  normal: {
    marginBottom: 4,
    borderRadius: 4,
    borderColor: BORDER_COLOR,
    borderWidth: 1,
  },
  error: {
    marginBottom: 4,
    borderRadius: 4,
    borderColor: ERROR_COLOR,
    borderWidth: 1,
  },
  open: {
    marginBottom: 4,
    borderRadius: 4,
    borderColor: 'blue',
    borderWidth: 1,
  }
}
```

However, if you use multiple "state" declarations, you can reduce the code to the following:

```javascript
pickerContainer: {
  normal: {
    marginBottom: 4,
    borderRadius: 4,
    borderColor: BORDER_COLOR,
    borderWidth: 1,
  },
  error: {
    borderColor: ERROR_COLOR,
  },
  open: {
    borderColor: OPEN_BORDER_COLOR,
  }
}
```

It's a personal preference and makes the stylesheets smaller and manageable.